src/pages/roadmap/index.tsx

```diff
@@ -18,86 +18,265 @@ type PeriodData = {
 };
 
 const roadmapData = {
-  '2022/Q3-Q4': {
+  '2022/Q4': {
+    future: false,
+    active: true,
+    items: [
+      {
+        name: 'CLI',
+        description: (
+          <p>
+            Command Line Interface will help users with extracting localization
+            data from the Tolgee platform, analyzing code, looking for new keys,
+            and extracting data in specific formats. All are customizable and
+            flexible!
```

Comment:
All are customizable and flexible - unclear for me. Maybe: Everything mentionned is customizable and flexible?